cmd/query/app/apiv3/grpc_handler_test.go

```diff
@@ -79,7 +79,50 @@ func newTestServerClient(t *testing.T) *testServerClient {
 
 func TestGetTrace(t *testing.T) {
 	tsc := newTestServerClient(t)
-	tsc.reader.On("GetTrace", matchContext, matchTraceID).Return(
+	traceIdLiteral := "156"
+	traceId, _ := model.TraceIDFromString(traceIdLiteral)
+
+	expectedTraceGetParameters := spanstore.GetTraceParameters{
+		TraceID:   traceId,
+		StartTime: time.Time{},
+		EndTime:   time.Time{},
+	}
+	tsc.reader.On("GetTrace", matchContext, expectedTraceGetParameters).Return(
```

Comment:
too many redundant code here. You can make the existing test to be table driven

Reply:
done